### ISSUE TITLE
chown errors when starting 3.6.1

### ISSUE THREAD

**Issue by nwl** [linuxserver/docker-sabnzbd#154]
## Expected Behavior
Container should start without throwing chown errors

## Current Behavior

Using 3.6.1, the logs show a failure to chown various directories. Downgrading to 3.6.0 and the container starts normally.
Unlike the issue at https://github.com/linuxserver/docker-sabnzbd/issues/151 , the config is not mounted on a network share but
on a single filesystem. To remove all permission-related issues, all the directories listed in docker-compose are set 777 and owned by the user/group listed in the compose file. 

## Steps to Reproduce

1. Run docker-compose
2. View docker ps and see container is unhealthy (unresponsive on port 8080)
3. Exec'ing into the container and /bin/bash hangs after receiving input

## Environment
**OS:** Ubuntu 22.04.1
**CPU architecture:** x86_64
**How docker service was installed:** lscr.io/linuxserver/sabnzbd:3.6.1

## Command used to create docker container (run/create/compose/screenshot)
  sabnzbd:
    image: lscr.io/linuxserver/sabnzbd:3.6.1
    volumes:
      - /home/nzb/sabnzbd/:/config
      - /home/nzb/monitor/:/config/monitor
      - /home/nzb/Downloads/complete/:/downloads
      - /home/nzb/Downloads/incomplete/:/incomplete-downloads
    ports:
      - "127.0.0.1:8080:8080"
    environment:
      - PGID=1006
      - PUID=1004
      - TZ=UTC
    healthcheck:
      test: curl --fail http://localhost:8080 || exit 1

## Docker logs
[custom-init] No custom services found, skipping...
[custom-init] Legacy service folder /config/custom-services.d is empty, deleting...
[migrations] started
[migrations] no migrations found

-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/

Brought to you by linuxserver.io
-------------------------------------

To support the app dev(s) visit:
SABnzbd: https://sabnzbd.org/donate

To support LSIO projects visit:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------

User uid:    1004
User gid:    1006
-------------------------------------

chown: changing ownership of '/config': Operation not permitted
chown: changing ownership of '/downloads': Operation not permitted
chown: changing ownership of '/incomplete-downloads': Operation not permitted
chown: changing ownership of '/config': Operation not permitted
chown: changing ownership of '/config/.bash_history': Operation not permitted
chown: changing ownership of '/config/logs': Operation not permitted
chown: changing ownership of '/config/logs/sabnzbd.log': Operation not permitted
chown: changing ownership of '/config/logs/sabnzbd.log.1': Operation not permitted
chown: changing ownership of '/config/logs/sabnzbd.log.3': Operation not permitted
chown: changing ownership of '/config/logs/sabnzbd.error.log': Operation not permitted
chown: changing ownership of '/config/logs/sabnzbd.log.2': Operation not permitted
chown: changing ownership of '/config/logs/sabnzbd.log.5': Operation not permitted
chown: changing ownership of '/config/logs/sabnzbd.log.4': Operation not permitted
chown: changing ownership of '/config/admin': Operation not permitted
chown: changing ownership of '/config/admin/rss_data.sab': Operation not permitted
chown: changing ownership of '/config/admin/history1.db': Operation not permitted
chown: changing ownership of '/config/admin/queue10.sab': Operation not permitted
chown: changing ownership of '/config/admin/watched_data2.sab': Operation not permitted
chown: changing ownership of '/config/admin/postproc2.sab': Operation not permitted
chown: changing ownership of '/config/admin/Rating.sab': Operation not permitted
chown: changing ownership of '/config/admin/totals10.sab': Operation not permitted
chown: changing ownership of '/config/monitor': Operation not permitted
s6-rc: warning: unable to start service init-sabnzbd-config: command exited 1

**Comment by aptalca** [comment]
how did you install the docker service?

**Comment by todaywasawesome** [comment]
Related to #151

**Comment by j0nnymoe** [comment]
@todaywasawesome no it's not. Specifically because I mentioned in that issue to this user to start a new issue.

**Comment by nwl** [comment]
Docker is installed from upstream Docker packages. Running:
docker-ce      5:20.10.20~3-0~ubuntu-jammy

**Comment by Honken77** [comment]
I have the same issue since I updated a few days ago. All my configs and downloads (sabnzbd, sonarr, radarr, jellyfin) lies on a NFS server. The share is setup with owner **nobody:nogroup** and the export is configured with **all_squash**. This works for all the apps but sabnzbd (which it did previously).

When I commented out my volume binds in the compose file for sabnzbd, it could start with a local volume on the docker server, so the problem is 100% related to the permissions on the folders. If I exec into the container as user **abc** I can create files in /config, but if I try to chown I get the same error as in the docker log. I don't see why a chown is needed, and I have no idea if that behaviour has changed in a recent build. I don't remember which image I used before I updated. I prune all images when I update.
